soc/microchip/pic32cxsg/common/Kconfig.pic32cxsg

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 Microchip
+
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_PIC32CXSG41 || SOC_SERIES_PIC32CXSG60 || SOC_SERIES_PIC32CXSG61
+
+config SOC_MICROCHIP_PIC32CXSG_XOSC32K
+	bool "The external 32 kHz crystal oscillator"
+	help
+	  Say y to enable the external 32 kHZ crystal oscillator at
+	  startup.  This can then be selected as the main clock source
+	  for the SOC.
```

Comment:
@nandojve to make it clear, this to dts, the rest can stay in Kconfig

Reply:
I see now what you want to avoid and I agreed with it.